src/main/java/picard/analysis/CollectWgsMetricsWithNonZeroCoverage.java

```diff
@@ -43,14 +45,16 @@
 import java.io.File;
 import java.util.List;
 
+@DocumentedFeature
+@BetaFeature
 @CommandLineProgramProperties(
         summary = CollectWgsMetricsWithNonZeroCoverage.USAGE_SUMMARY + CollectWgsMetricsWithNonZeroCoverage.USAGE_DETAILS,
         oneLineSummary = CollectWgsMetricsWithNonZeroCoverage.USAGE_SUMMARY,
         programGroup = Alpha.class
 )
 public class CollectWgsMetricsWithNonZeroCoverage extends CollectWgsMetrics {
 
-    static final String USAGE_SUMMARY = "Collect metrics about coverage and performance of whole genome sequencing (WGS) experiments.  ";
+    static final String USAGE_SUMMARY = "(Experimental) Collect metrics about coverage and performance of whole genome sequencing (WGS) experiments.  ";
```

Comment:
@sooheelee I think its a bit confusing to have tools that are categorized under "Alpha", tagged as "Beta", and prefixed with "Experimental", all at the same time. We do plan to add an "Experimental" annotation to Barlcay, similar to Beta, but its not yet implemented.

Reply:
I'm just matching GATK doc conventions. Pending Geraldine's approval, I hope to change the categorization in GATK so that there won't be this Alpha super-category, only functional categories, e.g. metrics collection, CNV discovery tools. The Alpha tools will be listed under its function category with the alpha label.

Reply:
@cmnbroad is the barclay "Experimental" annotation going to be done in the next month or after the Jan 9 release?

Reply:
I see. I like the idea of using program groups for functional categorization, and then using tags like BetaFeature or Experimental (but not both ?) to tag/assign a maturity level. Right now we have a mix. Alpha is a program group that isn't used directly by GATK, but it shows up in GATK since we include the Picard tools. "BetaFeature" is an annotation not used in Picard (until now); in GATK its used to prefix summaries with "(Beta) ", but Picard doesn't implement that. I think just adding a "BetaFeature" tag to a Picard tool with an Alpha program group will just add to the confusion.

I propose that we add the Experimental annotation to Barclay (I can do that pretty quickly, though we'll have to do a release to integrate it into GATK and Picard), and then we change both Picard and GATK to include the string prefixes for both on the summary lines. Then I think we'd have pretty good consistency in both Picard standalone and integrated GATK.

I'm not sure what the Picard "Unit Testing" category means, or if we need to do anything special for Deprecated.

Reply:
I'll bring these points up with Geraldine when I meet with her today.

Reply:
The testing category should go away -- I would say it's ok to replace with whatever the true functional group would be. 

Adding the (Experimental) prefix pending having the proper tag available is fine with me.